mealie/schema/recipe/recipe.py

```diff
@@ -141,10 +141,11 @@ class Recipe(RecipeSummary):
     nutrition: Optional[Nutrition]
 
     # Mealie Specific
-    settings: Optional[RecipeSettings] = RecipeSettings()
+    settings: Optional[RecipeSettings] = None
```

Comment:
Why was this change required for OCR?

Reply:
In short, giving the settings property of a Recipe a default value through RecipeSettings() meant it was not possible to create a recipe with custom settings.
This change is part of commit [90e1b85](https://github.com/hay-kot/mealie/pull/1244/commits/90e1b85135610b3364f5e2c190869123f5cbd042) where you will see that I also modified the `create_one` function of the recipe service to check if recipe settings are set, if not set them to the group's default.
Before that, when you passed a Recipe to this function, it would overwrite the settings with the group's defaults regardless if they were set or not.
To make it nice then, I set the default for the settings property to `None`, making sure that when a whole Recipe, we know whether someone put custom settings or whether we should use the group's defaults as defaults.

This change is part of this PR because I wanted to show assets by default on the recipes that are created using the OCR creation form. You will see in the diff of the above-mentioned commit that before making that change, I had to get the just created recipe via slug, set the custom setting and update the Recipe, which is less than sub-optimal.